pkg/discovery/publish.go

```diff
@@ -36,7 +46,7 @@ func (a *publisher) PublishLoop() {
 }
 
 // publish Publishes the API to Amplify Central.
-func (a *publisher) publish(serviceDetail *ServiceDetail) error {
+func (a *publishLoop) bbbpublish(serviceDetail *ServiceDetail) error {
```

Comment:
What does "bbb" represent in the prefix here?

Reply:
This was from a commit when I was testing. It's been removed